lib/transaction.py

```diff
@@ -425,8 +425,7 @@ def parse_input(vds):
 
 def parse_witness(vds):
     n = vds.read_compact_size()
-    for i in range(n):
-        x = vds.read_bytes(vds.read_compact_size())
+    return list(vds.read_bytes(vds.read_compact_size()).encode('hex') for i in xrange(n))
```

Comment:
It returned None before.  But the update_signature function needs the witnesses to update the signatures.

I guess update_signature isn't used if you sign with the seed in electrum; it is only used for hardware wallets.